app/src/main/java/nl/avans/drivioapp/view/AdvertisementDetailsFragment.kt

```diff
@@ -122,7 +122,8 @@ class AdvertisementDetailsFragment : Fragment(R.layout.fragment_advertisement_de
 
 
                 if (response?.code() == 200) {
```

Comment:
Misschien handig om deze response door te laten verwijzen naar een ander fragment (zelfde voor andere plekken waar de response is)

Reply:
Deze is aangepast om naar de bookings fragment te gaan

Reply:
Reverted omdat er anders falende ui testen komen